execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -1168,4 +1169,29 @@ where
         self.transforms = Vec::new();
         self
     }
+
+    /// Advances eras by num_eras
+    pub fn advance_eras_by(&mut self, num_eras: u64, reward_items: Vec<RewardItem>) {
```

Comment:
```suggestion
    pub fn advance_eras_by(&mut self, num_eras: u64, reward_items: impl IntoIterator<Item = RewardItem>) {
```

Why not this - in bunch of places I saw invocations like:

```rust
  .advance_era(vec![reward_item1, reward_item2])
```
We can make it flexible for users so they could just call it with array `advance_era([reward_item1, reward_item2])` if they have reward items created already

Reply:
Done in [`bc80b7d` (#2753)](https://github.com/casper-network/casper-node/pull/2753/commits/bc80b7dfcaba9b846c5d7e01aeef6f52e8042867)